hal/lib/athena/handles/HandlesInternal.h

```diff
@@ -23,6 +23,8 @@
 
 namespace hal {
 
+constexpr int16_t InvalidGetHandle = -1;
```

Comment:
Well the naming there might be confused with HAL_INVALID_HANDLE. That constant should only be used when calling a get index function. Would something like ```HandleGetValueFailed``` be better?